packages/dds/matrix/test/matrix.spec.ts

```diff
@@ -210,80 +210,16 @@ describe("Matrix", () => {
-            });
-
-            it("forEachCell 16x16 empty with blanks skipped", () => {
-                matrix.insertRows(0, 16);
-                matrix.insertCols(0, 16);
-                matrix.forEachCell(() => {
-                    assert.fail();
-                });
-            });
-
-            it("forEachCell 16x16 empty with blanks", () => {
-                matrix.insertRows(0, 16);
-                matrix.insertCols(0, 16);
-                let count = 0;
-                matrix.forEachCell((v) => {
-                    assert.equal(v, undefined);
-                    count++;
-                }, { includeEmpty: true });
-                assert.equal(count, 16 * 16);
-            });
-
-            it("forEachCell 16x16 empty with 1 cell", () => {
-                matrix.insertRows(0, 16);
-                matrix.insertCols(0, 16);
-                matrix.setCell(0, 0, -42);
-                let count = 0;
-                matrix.forEachCell((v) => {
-                    assert.equal(v, -42);
-                    count++;
-                });
-                assert.equal(count, 1);
-            });
-
-            it("forEachCell 16x16 empty with 1 cell and blanks", () => {
-                matrix.insertRows(0, 16);
-                matrix.insertCols(0, 16);
-                matrix.setCell(0, 0, -42);
-                let count = 0;
-                matrix.forEachCell((v, r, c) => {
-                    assert.equal(v, r === 0 && c === 0 ? -42 : undefined);
-                    count++;
-                }, { includeEmpty: true });
-                assert.equal(count, 16 * 16);
+                fill(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 256, /* colCount: */ 256);
+                check(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 256, /* colCount: */ 256);
             });
         });
 
         describe("fragmented", () => {
-            it("read/write 16x16", () => {
-                insertFragmented(matrix, 16, 16);
-                fill(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 16, /* colCount: */ 16);
-                check(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 16, /* colCount: */ 16);
-            });
-        });
-
-        describe("annotations", () => {
-            it("read/write 16x16", () => {
-                matrix.insertRows(0, 256);
-                matrix.insertCols(0, 256);
-                matrix.setAnnotation(0, 0, -42);
-                assert.equal(matrix.getAnnotation(0, 0), -42);
-                matrix.setCell(0, 0, 1);
-                // Setting a cell should clear existing annotation.
-                assert.equal(matrix.getAnnotation(0, 0), undefined);
+            it("read/write 256x256", () => {
+                insertFragmented(matrix, 256, 256);
+                fill(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 256, /* colCount: */ 256);
+                check(matrix, /* row: */ 0, /* col: */ 0, /* rowCount: */ 256, /* colCount: */ 256);
```

Comment:
Note: This used to be only 16x16 due to poor performance with fragmented matrices.  With the handle cache work we did way back, this is pretty much just as fast as the contiguous case.